backend/main.py

```
@@ -0,0 +1,48 @@
+@app.get("/")
+async def root():
+    return {"message": "OCR service is running"}
+
+@app.post("/azure")
+async def evaluate(url: str = Query(...)):
+    # Create a temporary file
+    temp_filename = 'temp.jpg'
+
+    # Download the image
+    def download_image():
+        response = requests.get(url)
```

Ruff (0.14.8)

Probable use of `requests` call without timeout

(S113)